src/styles.scss

```diff
@@ -551,6 +555,14 @@ div.button a {
   margin-top: 8px;
 }
 
+.mat-raised-button:hover {
+  text-decoration: none;
```

Comment:
it should already be none. Is there something else causing it to not be? Find and remove that instead

Reply:
My mistake, that shouldn't be there. Forgot to remove it after testing. Removed now